stdlib/2/__builtin__.pyi

```diff
@@ -30,26 +30,27 @@ class classmethod: pass  # Special, only valid as a decorator.
 class object:
     __doc__ = ...  # type: Optional[str]
     __class__ = ...  # type: type
+    __dict__ = ...  # type: Dict[str, Any]
     __slots__ = ...  # type: Optional[Union[str, unicode, Iterable[Union[str, unicode]]]]
+    __module__ = ...  # type: Any
 
     def __init__(self) -> None: ...
     def __new__(cls) -> Any: ...
     def __setattr__(self, name: str, value: Any) -> None: ...
-    def __eq__(self, o: object) -> bool: ...
-    def __ne__(self, o: object) -> bool: ...
```

Comment:
Ah, good catch. Looks like a Python 2/3 compatibility issues -- in Python 3 they exist:

```
>>> object().__eq__
<method-wrapper '__eq__' of object object at 0x10062e110>
```